src/cljs/athens/util.cljs

```diff
@@ -92,3 +92,16 @@
       (string/replace x #"PM" "pm"))))
 
 
+;; -- Regex -----------------------------------------------------------
+
+;; https://stackoverflow.com/a/11672480
+(def regex-esc-char-map
+  (let [esc-chars "()*&^%$#![]"]
+    (zipmap esc-chars
+            (map #(str "\\" %) esc-chars))))
+
+
+(defn escape-str
+  "Take a string and escape all regex special characters in it"
+  [str]
+  (string/escape str regex-esc-char-map))
```

Comment:
Just a FYI, Clj is both nice and not nice because you can shadow/override clojure.core symbols. In this closure, you are shadowing `str` with an input parameter, so if you tried to use `str` the clojure.core function like `(str "hello" "world")` you would get a function invocation error.

Reply:
I had some difficulty tracking some bugs in `keybindings` because I bound a variable with the symbol `key`, which is also a clojure.core function!